x/sequencers/client/cli/tx.go

```diff
@@ -0,0 +1,133 @@
+package cli
+
+import (
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+
+	"github.com/dymensionxyz/dymension-rdk/x/sequencers/types"
+)
+
+// NewCmdGrantPermissionsProposal broadcasts a GrantPermissionsProposal message.
+func NewCmdGrantPermissionsProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "grant-permissions-proposal address module-name [flags]",
+		Short:   "proposal to grant permissions for a specific address",
+		Example: `dymd tx gov submit-legacy-proposal grant-permissions-proposal address module_name_1,module_name_2,...`,
```

Comment:
the permissions are just generic strings right, not module names?

Reply:
Permissions are modules's name, cause that's what I set checking in hub-genesis and denommetadata

Reply:
I see, but the sequencer module doesn't know that